cordyceps/src/sorted_list.rs

```diff
@@ -0,0 +1,459 @@
+//! [Intrusive], singly-linked, sorted, linked-list.
+//!
+//! See the documentation for the [`SortedList`] and [`SortedListIter`] types for
+//! details.
+//!
+//! [intrusive]: crate#intrusive-data-structures
```

Comment:
nit: i _think_ this link is broken, because you capitalized it above...
```suggestion
//! [Intrusive]: crate#intrusive-data-structures
```